console/console.properties

```diff
@@ -79,9 +80,12 @@ pendingOrgSearchBaseDN=ou=pendingorgs
 psql.url=jdbc:postgresql://localhost:5432/georchestra
 psql.user=georchestra
 psql.pass=georchestra
-dataSource.minPoolSize=1
-dataSource.maxPoolSize=10
-dataSource.timeout=2000
+dataSource.minPoolSize = 2
```

Comment:
Is there a particular reason for bumping `minPoolSize` to 2 ?

Reply:
It was the default value in the hardcoded properties files we removed (see https://github.com/georchestra/georchestra/pull/2508/files#diff-bcd5b2434945223adb44a0faaa110ed8L71).
And that value has been set two weeks ago: https://github.com/georchestra/georchestra/commit/ccce93a21e507162e1e4b45c17c43982bd641654

Reply:
okay !